### PR TITLE
Document/update readme

### PR DESCRIPTION
**Pull request by mikeslgoh**
Updated the README. Please help suggest things to add

### CODE REVIEW

**Comment by mikeslgoh** [comment]
[Update README on Github with project description](https://trello.com/c/H6tiajCc/141-update-readme-on-github-with-project-description)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+____________________________
+### Features:
+- Park in the most amount of spots as fast as you can
+- Keyboard and Controller support for gameplay and UI
+- Bumper car physics
+_________
+### Plans for Upcoming Features:
+- 3D art style
+- MORE ORIGINAL MUSIC
+- New game modes
```
Comment (Gursaanj) [inline README.md]: Should we go like "New Game Modes... Such As"

File: README.md [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+- Better player customization
+- Power ups
+____________________________
```
Comment (Gursaanj) [inline README.md]: Other Upcoming Features:

- More Maps / Better Maps 
- NPC/AI integration
- Adjustable Settings (Settings feature)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+
+Do you have what it takes to master the art of parking?
+____________________________
+### Features:
+- Park in the most amount of spots as fast as you can
+- Keyboard and Controller support for gameplay and UI
+- Bumper car physics
+_________
+### Plans for Upcoming Features:
+- 3D art style
```
Comment (Gursaanj) [inline README.md]: When Ready, maybe put a screenshot of the 3d art style (like concept art or something)

File: README.md [hunk 4]
```diff
@@ -1,4 +1,20 @@
...
+# BEEP BEEP
 
-Description of the project & scope will go HERE
+### Description:
+Local multiplayer bumper cars meets Crazy Taxi ... have fun with friends without the lag. 
+
+Do you have what it takes to master the art of parking?
+____________________________
+### Features:
+- Park in the most amount of spots as fast as you can
```
Comment (Gursaanj) [inline README.md]: Not sure if this should be in Features?, Maybe place in Description

Comment (m4rtint) [reply README.md]: Agree

File: README.md (hunk 3 above)
Comment (mikeslgoh) [reply README.md]: I tried putting the video I had on it but README on Github doesn't really work

Comment (Gursaanj) [reply README.md]: https://stackoverflow.com/questions/34341808/is-there-a-way-to-add-a-gif-to-a-markdown-file

Does this help,or is specifically the ReadMe file unable to play it

**Pull request merged**